### DISCUSSION TITLE
Aderyn issue while running `aderyn .`

### DISCUSSION THREAD

**Discussion by harshraj070** [Cyfrin/security-and-auditing-full-course-s23#272] [General]
Whenever I try to run the Aderyn commands, i get this error-

![Screenshot 2024-12-28 193956](https://github.com/user-attachments/assets/4af32afe-25c0-4d1d-a179-c5609ed09f30)

**Comment by EngrPips** [comment]
Hello @harshraj070, Please use the official Cyfrin Discord https://discord.gg/qUEBFM2Y to seek help regarding this

**Comment by Keivou** [comment]
Did you find out what it is @harshraj070 ? I'm having the same problem. I tried to modify the `lib.rs` file to remove the `None` values, but nothing, the error persists.

**Reply by EngrPips** [reply]
You can use the official Cyfrin Discord https://discord.gg/qUEBFM2Y to seek help regarding this

**Comment by harshraj070** [comment]
nope. I'm just using slither instead.
